src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -3582,7 +3614,7 @@ static std::unordered_map<uint32_t, GfxOpcodeHandlerFunc> f3dex2Handlers = {
     { G_PUSHCD, gfx_pushcd_handler_f3dex2 },
     { G_BRANCH_Z_OTR, gfx_branch_z_otr_handler_f3dex2 },
     { G_ENDDL, gfx_end_dl_handler_common },
-    { G_SETPRIMDEPTH, gfx_set_prim_depth_handler_f3dex2 },
+    { G_SETPRIMDEPTH, gfx_set_prim_depth_handler_rdp },
```

Comment:
Do we need `G_SETPRIMDEPTH` in f3dex2 handlers if it is already declared in the rdp handlers?